mmv1/products/dialogflowcx/api.yaml

```diff
@@ -560,3 +560,105 @@ objects:
           - :RUNNING
           - :SUCCEEDED
           - :FAILED
+  - !ruby/object:Api::Resource
+    name: 'EntityType'
+    base_url: "{{parent}}/entityTypes"
```

Comment:
Is there a reason to prefer `{{parent}}` here rather than `projects/{{project}}/locations/{{location}}/agents/{{agent}}`?

In general I prefer expanding this to the longer form, as it allows more options for users to inherit the project from their provider default.

Reply:
It will also make the custom import not needed

Reply:
hi @slevenick the problem is that **location** is not an attribute of the resource, so when the replace of the url is executed it fails. Is there a way to make the replace work?
`url, err := replaceVars(d, config,{{DialogflowCXBasePath}}projects/{{project}}/locations/{{location}}/agents/{{agent}}")
`
I see the logic of the replaceVars and it seems is only using the properties of the resource

Reply:
Well, we could add `location` as a property of the resource and set it to `url_param_only` or something like that to prevent it from being sent in the request body. I see that other dialogflowcx resources are already using `{{parent}} so it may be better to keep these consistent

Reply:
sure thanks